File: app/src/main/java/com/wismna/geoffroy/donext/data/TaskList.java
package com.wismna.geoffroy.donext.data;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "tasklist")
public class TaskList {

    @PrimaryKey(autoGenerate = true)
    public long _id;

    @ColumnInfo(name = "name")
    public String name;

    @ColumnInfo(name = "visible")
    public boolean visible = true;

    @ColumnInfo(name = "displayorder")
    public int order;

    //@ColumnInfo(name = "taskcount")
    public int taskCount;
}

File: app/src/main/java/com/wismna/geoffroy/donext/data/TaskList.kt
package com.wismna.geoffroy.donext.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "tasklist")
class TaskList {
    @JvmField
    @PrimaryKey(autoGenerate = true)
    var _id: Long = 0

    @JvmField
    @ColumnInfo(name = "name")
    var name: String? = null

    @JvmField
    @ColumnInfo(name = "visible")
    var visible = true

    @JvmField
    @ColumnInfo(name = "displayorder")
    var order = 0

    //@ColumnInfo(name = "taskcount")
    @JvmField
    var taskCount = 0
}
